.eleventy.js

```diff
@@ -49,19 +49,23 @@ module.exports = (config) => {
     lstripBlocks: true,
   })
   config.setLibrary('njk', nunjucksEnv)
-  config.addNunjucksFilter('rejectattr_path', (array, propertyPath) => (
+
+  const markdownIt = new MarkdownIt()
+  config.addFilter('markdown', (markdown) => markdown && new SafeString(markdownIt.render(markdown)))
```

Comment:
Couldn't reuse the Eleventy instance of markdown-it in a nice way so had to create another one here.